nvm.sh

```diff
@@ -1231,6 +1231,29 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  echo $NVM_CPU_THREADS | egrep -q '^[0-9]{1,}$' &> /dev/null
+  thread_valid=$?
```

Comment:
please declare this var with `local` first, and also please follow the all-caps `NVM_`-prefixed convention to account for `ksh`, which doesn't support `local`.